### PR TITLE
ref(ts) Convert Organization Project List to typescript

### PR DESCRIPTION
**Pull request by markstory**
This should help make the chart replacement easier.

### CODE REVIEW

**Review by scttcper — APPROVED** [review]
good stuff, we should add the types for react-lazyload

File: src/sentry/static/sentry/app/views/settings/organizationProjects/index.tsx [hunk 1]
```diff
@@ -12,43 +15,46 @@ import Pagination from 'app/components/pagination';
...
+type State = AsyncView['state'] & {
+  projectList: Project[] | null;
+  projectListPageLinks: string | null;
+  projectStats: ProjectStats | null;
+};
+
+class OrganizationProjects extends AsyncView<Props, State> {
   static contextTypes = {
-    router: PropTypes.object.isRequired,
+    router: PropTypes.object,
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/settings/organizationProjects/index.tsx:44]: do we still need this contextTypes? it doesn't seem to be being used

**Review by priscilawebdev — COMMENTED** [review]
@markstory it's not visible, but please as you are touching this code, maybe you could replace the `div` that wrappers this component with `React.ReactFragment`? and maybe replace the css in the PanelBody `<PanelBody css={{width: '100%'}}>` with emotion?

**Comment by markstory** [comment]
> it's not visible, but please as you are touching this code, maybe you could replace the div that wrappers this component with React.ReactFragment? and maybe replace the css in the PanelBody <PanelBody css={{width: '100%'}}> with emotion?

I can do that in the next pull request I am going to do which will also swap the chart out with the echarts based one.

File: src/sentry/static/sentry/app/views/settings/organizationProjects/index.tsx (hunk 1 above)
Comment (markstory) [reply src/sentry/static/sentry/app/views/settings/organizationProjects/index.tsx:44]: I'll remove it in that follow up pull request.

**Pull request merged**